### PR TITLE
Install root dependencies

### LINKED ISSUE #181: installed packages do not seem to be included in output
Trying to track down errors we are seeing in deploys on Modulus. Here is an example that shows that maybe `meteor build` is not honoring the root `package.json` dependencies...?

```
[~/sample-apps/meteor]
› meteor create simple-todos
Created a new Meteor app in 'simple-todos'.

To run your new app:
  cd simple-todos
  meteor

If you are new to Meteor, try some of the learning resources here:
  https://www.meteor.com/learn


[~/sample-apps/meteor]
› cd simple-todos

[~/sample-apps/meteor/simple-todos]
› meteor npm i left-pad
left-pad@1.0.2 node_modules/left-pad

[~/sample-apps/meteor/simple-todos]
› cat package.json
{
  "name": "simple-todos",
  "private": true,
  "scripts": {
    "start": "meteor run"
  },
  "dependencies": {
    "left-pad": "~1.0.2",
    "meteor-node-stubs": "~0.2.0"
  }
}

[~/sample-apps/meteor/simple-todos]
› demeteorizer
Demeteorizing application...

WARNING: The output directory is under your source tree.
         Your generated files may get interpreted as source code!
         Consider building into a different directory instead
         meteor build ../output

Demeteorization complete.

[~/sample-apps/meteor/simple-todos]
› cd .demeteorized

[~/sample-apps/meteor/simple-todos/.demeteorized]
› ag left-pad
```

`left-pad` is nowhere to be found in my output...

### PR DESCRIPTION
**Pull request by theworkflow**
Install root dependencies when `package.json` is present. Then moves them to the `demeteorized` bundle.

Closes #181

### CODE REVIEW

File: lib/move-dependencies.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  var source = Path.resolve(options.input, 'node_modules');
+  var destination = Path.resolve(options.directory,
+    'bundle/programs/server/node_modules'
+  );
+
+  FS.readdir(source, function (err) {
+    if (err && err.code === 'ENOENT') return done();
+    if (err) return done(err);
+
+    FS.copy(source, destination, done);
```
Comment (fiveisprime) [inline lib/move-dependencies.js]: Should this be called `copy-dependencies.js` instead? 🙃

File: package.json [hunk 2]
```diff
@@ -24,8 +24,9 @@
   },
   "dependencies": {
     "commander": "2.9.0",
-    "hoek": "4.x.x",
-    "cross-spawn": "3.0.1"
+    "cross-spawn": "3.0.1",
+    "fs-extra": "^0.30.0",
```
Comment (fiveisprime) [inline package.json]: Remove the `^`

**Comment by fiveisprime** [comment]
I fixed the build issues and resolved the conflict with `master` since I broke that..

**Comment by theworkflow** [comment]
Merge conflicts have been resolved.

**Comment by jackboberg** [comment]
@theworkflow Are you considering closing #243 with this PR?

**Review by fiveisprime — APPROVED** [review]

**Comment by theworkflow** [comment]
@jackboberg Not initially. I had the thought process that each function/file is an individual task. Updating the `engines` block of the `package.json` would be a different task. If you feel it should be handled best in this PR I'm down for adding it.

**Review by jackboberg — APPROVED** [review]
A separate PR seems fine, I think it could be a part of [`find-node-version`](https://github.com/onmodulus/demeteorizer/blob/master/lib/find-node-version.js).

**Pull request merged**